test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -4195,6 +4196,20 @@ TEST_F(AtenXlaTensorTest, TestRandperm) {
                           cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestRandpermOut) {
+  int n = 5;
+  torch::Tensor shuffle;
+  torch::randperm_out(shuffle, n);
+  // xw32: why do we need to copy tensor "shuffle" to CPU?
+  torch::Tensor shuffle_cpu = CopyToDevice(shuffle, torch::kCPU);
```

Comment:
Because torch::RandPerm "Returns a random permutation of integers from 0 to n - 1", there is no guarantee that the native pytorch and torch-xla return the same permutation. So I follow the existing test "TestRandperm" which only runs it on one device and validate the result is a valid permutation.